data/pathfinder/alluria_publishing/remarkable_races/rr_races.lst

```diff
@@ -34,7 +34,7 @@ Purple Peeper									STARTFEATS:1	SIZE:T
 Battle Woggart									STARTFEATS:1	SIZE:L	MOVE:Walk,40,Swim,40																																					BONUS:STAT|STR|8	BONUS:STAT|DEX|4	BONUS:STAT|CON|6	BONUS:STAT|INT|-8	BONUS:STAT|WIS|2	BONUS:STAT|CHA|-4																									BONUS:WEAPONPROF=Bite|DAMAGESIZE|1													ABILITY:Internal|AUTOMATIC|Bite|Race Traits ~ Woggart								LEGS:4	HANDS:2																				MONSTERCLASS:Animal:5				RACETYPE:Animal				RACESUBTYPE:Amphibian							TYPE:Animal			CR:3			SOURCEPAGE:p.144
 Dwarf Battle Woggart								STARTFEATS:1	SIZE:L	MOVE:Walk,40,Swim,40																																					BONUS:STAT|STR|4	BONUS:STAT|DEX|4	BONUS:STAT|CON|4	BONUS:STAT|INT|-8	BONUS:STAT|WIS|2	BONUS:STAT|CHA|-4																									BONUS:WEAPONPROF=Bite|DAMAGESIZE|1													ABILITY:Internal|AUTOMATIC|Bite|Race Traits ~ Woggart								LEGS:4	HANDS:2																				MONSTERCLASS:Animal:5				RACETYPE:Animal				RACESUBTYPE:Amphibian							TYPE:Animal			CR:3			SOURCEPAGE:p.144
 Riding Woggart									STARTFEATS:1	SIZE:L	MOVE:Walk,50,Swim,50																																					BONUS:STAT|STR|6	BONUS:STAT|DEX|4	BONUS:STAT|CON|6	BONUS:STAT|INT|-8	BONUS:STAT|WIS|2	BONUS:STAT|CHA|-6																									BONUS:WEAPONPROF=Bite|TOHIT|-5	BONUS:WEAPONPROF=Bite|DAMAGE|-(STR/2)						ABILITY:Internal|AUTOMATIC|Bite|Race Traits ~ Riding Woggart						LEGS:4	HANDS:2																				MONSTERCLASS:Animal:5				RACETYPE:Animal				RACESUBTYPE:Amphibian							TYPE:Animal			CR:3			SOURCEPAGE:p.144
-Coin Drake										STARTFEATS:1	SIZE:L	MOVE:Walk,30,Burrow,30,Fly,100	FACE:10	REACH:10																								BONUS:COMBAT|AC|20|TYPE=NaturalArmor																			BONUS:VAR|OverrideRaceReach,RaceSizeIsLong|1	BONUS:VAR|Maneuverability|2																														ABILITY:Internal|AUTOMATIC|Race Traits ~ Coin Drake																										SR:21					MONSTERCLASS:Dragon:13				RACETYPE:Dragon														TYPE:Dragon			CR:10			SOURCEPAGE:p.146									KIT:1|TN
+Coin Drake										STARTFEATS:1	SIZE:L	MOVE:Walk,30,Burrow,30,Fly,100																									BONUS:COMBAT|AC|20|TYPE=NaturalArmor																			BONUS:VAR|OverrideRaceReach,RaceSizeIsLong|1	BONUS:VAR|Maneuverability|2																														ABILITY:Internal|AUTOMATIC|Race Traits ~ Coin Drake																										SR:21					MONSTERCLASS:Dragon:13				RACETYPE:Dragon														TYPE:Dragon			CR:10			SOURCEPAGE:p.146									KIT:1|TN
```

Comment:
Still "RaceSizeIsLong" in the file.  You should do a search in eclipse for the string in the data files.  While you're at it, look for "VAR|SizeIsLong" since there also seems to be a misnamed variable in the old system.